server/gameObjects/Player.js

```diff
@@ -94,6 +102,27 @@ class Player {
     this.toRespawn = false;
   }
 
+  addToBackpack(newItem, action) {
+    const item = this.backpack.items.find((i) => i.name === newItem);
+
+    if (item) {
+      item.quantity += 1;
+      this.energyUse(action);
```

Comment:
Not sure if this place is best for energy burn as not all actions will be backpack related, move to socket emit?